src/components/AddPeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -21,7 +21,7 @@
         <span>
           <v-icon color="blue darken-2">mdi-information-outline</v-icon>
           This step is not complete.
-          <router-link id="router-link" :to="{ path: '/add-people-roles', query: { showErrors: true } }">
+          <router-link id="router-link" :to="{ path: '/add-people-and-roles', query: { showErrors: true } }">
```

Comment:
All the more reason to have a (separate) enum for this ;)

Reply:
Created!

Reply:
Oh wait, we do now have 2 enums. Does it make sense to use RouteNames.XXX here?

Reply:
Yea, the route enum existed. I'm sure it just wasn't applied in this instance and most likely other `path` references.
It does for sure, but these "clean up refactors" can also wait until the framework ticket. I believe this ticket has already gone outside it's scope 😛